integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java

```diff
@@ -0,0 +1,147 @@
+        "insert into root.sg1.d2(time, s3, s4) values(21, 21, TRUE)",
+        "insert into root.sg1.d2(time, s3, s4) values(22, 22, TRUE)",
+        "insert into root.sg1.d2(time, s3, s4) values(23, 23, TRUE)",
+        "insert into root.sg1.d2(time, s3, s4) values(24, 24, TRUE)",
+        "insert into root.sg1.d2(time, s3, s4) values(25, 25, TRUE)",
+        "insert into root.sg1.d2(time, s3, s4) values(26, 26, FALSE)",
+        "insert into root.sg1.d2(time, s3, s4) values(27, 27, FALSE)",
+        "insert into root.sg1.d2(time, s3, s4) values(28, 28, FALSE)",
+        "insert into root.sg1.d2(time, s3, s4) values(29, 29, FALSE)",
+        "insert into root.sg1.d2(time, s3, s4) values(30, 30, FALSE)",
+        //        "flush",
+        "insert into root.sg1.d1(time, s1, s3, s4) aligned values(23, 230000.0, 230000, FALSE)",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(31, 31, 'aligned_test31')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(32, 32, 'aligned_test32')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(33, 33, 'aligned_test33')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(34, 34, 'aligned_test34')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(35, 35, 'aligned_test35')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(36, 36, 'aligned_test36')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(37, 37, 'aligned_test37')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(38, 38, 'aligned_test38')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(39, 39, 'aligned_test39')",
+        "insert into root.sg1.d1(time, s2, s5) aligned values(40, 40, 'aligned_test40')",
+        "insert into root.sg1.d2(time, s2, s5) values(31, 31, 'non_aligned_test31')",
+        "insert into root.sg1.d2(time, s2, s5) values(32, 32, 'non_aligned_test32')",
+        "insert into root.sg1.d2(time, s2, s5) values(33, 33, 'non_aligned_test33')",
+        "insert into root.sg1.d2(time, s2, s5) values(34, 34, 'non_aligned_test34')",
+        "insert into root.sg1.d2(time, s2, s5) values(35, 35, 'non_aligned_test35')",
+        "insert into root.sg1.d2(time, s2, s5) values(36, 36, 'non_aligned_test36')",
+        "insert into root.sg1.d2(time, s2, s5) values(37, 37, 'non_aligned_test37')",
+        "insert into root.sg1.d2(time, s2, s5) values(38, 38, 'non_aligned_test38')",
+        "insert into root.sg1.d2(time, s2, s5) values(39, 39, 'non_aligned_test39')",
+        "insert into root.sg1.d2(time, s2, s5) values(40, 40, 'non_aligned_test40')",
+      };
+
+  public static void insertData() throws ClassNotFoundException {
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+
+      // create aligned and non-aligned time series
+      for (String sql : sqls) {
+        statement.execute(sql);
```

Comment:
Please use `executerBatch` instead to accelerate the insert procedure.

Reply:
`executeBatch` has not been implemented in new cluster yet.

Reply:
So just add TODO comments for future

Reply:
ok